include/secp256k1_schnorrsig.h

```diff
@@ -175,6 +175,22 @@ SECP256K1_API SECP256K1_WARN_UNUSED_RESULT int secp256k1_schnorrsig_verify(
     const secp256k1_xonly_pubkey *pubkey
 ) SECP256K1_ARG_NONNULL(1) SECP256K1_ARG_NONNULL(2) SECP256K1_ARG_NONNULL(5);
 
+/** Verify the set of schnorr signatures or tweaked pubkeys present in the secp256k1_batch_context.
+ * 
+ *  Returns: 1: correct schnorrsigs/tweaks
+ *           0: incorrect schnorrsigs/tweaks
+ * 
+ *  In particular, returns 0 if the batch context is empty or NULL.
```

Comment:
is it okay if the `batch_context_verify` return 0 when the batch context is empty or `NULL`? In the previous implementation, it used to return 1 when `n_sigs = 0` ([link here](https://github.com/bitcoin-core/secp256k1/pull/760/files#diff-75af67a78d848e432dcf1b975d64978c8d731530fcbde6c9b15edf7da556af00R109)).

Reply:
I think either way is fine as long as it's documented. I'm leaning towards your solution now because it's slightly harder to mess up, e.g. in code like this:
```
void func(batch_I_actually_want_to_verify) {
  /* something something */
  return batch_verify(accidental_empty_batch)
}
```
But that's a pretty weak argument I think because if that happens, other bad stuff could happen as well.

Reply:
Now that I think about it. Empty batch won't be possible due to the `VERIFY_CHECK(n_terms > 0)` check during `batch_create`. The user also can't pass `NULL` instead of a batch_ctx pointer due to `SECP256K1_ARG_NONNULL`.

Reply:
> Empty batch won't be possible due to the `VERIFY_CHECK(n_terms > 0)` check during `batch_create`.

This is not true. If the user does:
```
batch *batch_ctx = batch_create(ctx, 3) /* here, n_terms > 0 */
batch_verify(ctx, batch_ctx) /* batch object is still empty when the user calls verify */
```